example/sign/lib/widgets/session_widget.dart

```diff
@@ -81,65 +71,36 @@ class SessionWidgetState extends State<SessionWidget> {
         chainMetadata.w3mChainInfo.chainName,
         style: StyleConstants.subtitleText,
       ),
-      const SizedBox(
-        height: StyleConstants.linear8,
-      ),
-      Text(
-        account,
-        textAlign: TextAlign.center,
-      ),
-      const SizedBox(
-        height: StyleConstants.linear8,
-      ),
-      const Text(
-        StringConstants.methods,
-        style: StyleConstants.subtitleText,
-      ),
+      const SizedBox(height: StyleConstants.linear8),
+      Text(account, textAlign: TextAlign.center),
+      const SizedBox(height: StyleConstants.linear8),
+      const Text(StringConstants.methods, style: StyleConstants.subtitleText),
     ];
 
-    children.addAll(
-      _buildChainMethodButtons(
-        chainMetadata,
-        account,
-      ),
-    );
+    children.addAll(_buildChainMethodButtons(chainMetadata, account));
 
     children.addAll([
-      const SizedBox(
-        height: StyleConstants.linear8,
-      ),
-      const Text(
-        StringConstants.events,
-        style: StyleConstants.subtitleText,
-      ),
+      const SizedBox(height: StyleConstants.linear8),
+      const Text(StringConstants.events, style: StyleConstants.subtitleText),
     ]);
-    children.addAll(
-      _buildChainEventsTiles(
-        chainMetadata,
-      ),
-    );
+    children.addAll(_buildChainEventsTiles(chainMetadata));
 
     // final ChainMetadata
     return Container(
-      width: double.infinity,
+      // width: double.infinity,
```

Comment:
Needed comments?

Reply:
I'll go over the comments now but everything inside `example/` folder will be changed and mostly removed as that is the example app.
Regarding the svg asset, good catch! That one was exported prior my work, I'll re-export it.

Reply:
Fixed. Compass asset looks better but still weird, not sure why but probably export issues...?